action.yml

```
@@ -11,9 +11,13 @@ inputs:
     description: 'GoReleaser version'
     default: 'latest'
     required: false
+  install-only:
+    description: 'Just install GoReleaser'
+    default: false
```

```suggestion
    default: 'false'
```

[`default`](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault) metadata must be a string here.